integration/tests/cook/test_basic.py

```diff
@@ -994,16 +994,60 @@ def check_unique_constraint(unscheduled_jobs):
         finally:
             util.kill_jobs(self.cook_url, [uuids])
 
+    def test_balanced_host_constraint_cannot_place(self):
+        state = util.get_mesos_state(self.mesos_url)
+        num_hosts = len(state['slaves'])
+        if num_hosts > 10:
+            # Skip this test on large clusters
```

Comment:
Nitpick, can we add a log specifying we are skipping this test and also mention the cluster size

Reply:
Done